assets/css/less/wpbdp/abstracts/mixins/mixins.less

```diff
@@ -0,0 +1,5 @@
+.flex-center {
```

Comment:
Can we use our wpdb- prefix for this to avoid conflicts?

Reply:
@stephywells This is a mixin, which is a reusable set of CSS properties that can be defined and called within a stylesheet using the Less. When this mixin is used, the Grunt task runner will remove the `.flex-center` class and apply its properties directly to the relevant HTML elements.

Reply:
Oh, got it. Thank you! 

It's showing in the minified css.

Reply:
@shervElmi looks like this hasn't been addressed yet. I'm not sure that I can test any other design PRs until this one is finalized.